book/09-git-and-other-scms/sections/client-bzr.asc

```diff
@@ -0,0 +1,102 @@
+==== Git and Bazaar
```

Comment:
What do you think about this introduction ?
> Among the DVCS, another famous one is [Bazaar](http://bazaar.canonical.com/en/).
> Bazaar is free and open source, and is part of the [GNU Project](http://www.gnu.org/).
> It behaves very differently from Git.
> Sometimes, to do the same thing as with Git, you have to use a different keyword, and some keywords that are common don't have the same meaning.
> The branch management is very different and may cause confusion, especially when someone comes from Git's universe.
> But strangely and fortunately, it is simple to switch from Bazaar to Git.

I made an adaption of the URLs for the format for markdown but I'll use the appropriate one for asciidoctor.